data/array/mapper/internal/7d61f7c0-a55f-4c1c-80ff-2b6fef8b70c4.json

```diff
@@ -37,8 +37,9 @@
       }
     }
   },
-  "name": "data/array/Mapper/Internal",
+  "name": "data/array/Mapper/internal",
```

Comment:
Is the lowercase intentional? All other internal nodes are called "Internal" with uppercase "I" at the moment.